deployments/epinio.go

```diff
@@ -244,3 +305,14 @@ func (k *Epinio) createIngress(c *kubernetes.Cluster, subdomain string) error {
 
 	return err
 }
+
+// hashApr1 generates an Apache MD5 hash for a password.
```

Comment:
md5 is not secure as encryption for passwords but we do store this in a secret. I'm not sure if it's worth it switching to SHA1 or bcrypt (both supported: https://doc.traefik.io/traefik/middlewares/basicauth/#configuration-options).
If someone can read the md5 hash, it means they can read the k8s secrets, they can probably do more then without needing the password itself to talk to the epinio API. 

otoh, generating a sha1 sum is anything but difficult: https://golang.org/pkg/crypto/sha1/#New so why not?

Reply:
[I read in various places](https://codahale.com/how-to-safely-store-a-password/) that bcrypt is the safest option but it's very slow (0.3 secs to hash a password?! though it depends on the work factor). That could make every request slower by 300ms which is not a negligible delay. Let's stick to MD5 for now. Securing k8s secrets is the right way to protect this password. After all, we store it in plain text on the cli side.